montblanc/__init__.py

```diff
@@ -23,7 +23,9 @@
 
 from montblanc.logsetup import setup_logging, setup_test_logging
 from montblanc.tests import test
-
+import tensorflow as tf
+os.environ["LD_LIBRARY_PATH"] = ":".join([tf.sysconfig.get_lib(), 
+                                          os.environ.get("LD_LIBRARY_PATH", "")])
```

Comment:
Why is it necessary to insert tensorflow's library path here? I guess its to allow loading of some shared library.

Could doing something like `import tensorflow.somelib` have the same effect?

Reply:
Linkage against the main tensorflow library seems to be failing on some tensorflow versions when the montblanc rime.so is loaded. I have since replaced the linker flag section so it may just work, I will try taking this back out and see if the linkage still works